conf/app.ini

```diff
@@ -19,6 +19,7 @@ use: call:h.app:create_app
 # If not provided, both default to a random URL-safe base64-encoded string.
 #h.client_id:
 #h.client_secret:
+h.client_url: https://unpkg.com/hypothesis
```

Comment:
We're slowly but surely phasing out use of these configuration files, so perhaps we could put the default value in an appropriate module in the application?